kafka/channel/README.md

```diff
@@ -29,16 +35,40 @@ topics.
      namespace: knative-eventing
    data:
      # Broker URL. Replace this with the URLs for your kafka cluster,
-     # which is in the format of my-cluster-kafka-bootstrap.my-kafka-namespace:9092.
+     # which is in the format of my-cluster-kafka-bootstrap .my-kafka-namespace:9092.
      bootstrapServers: REPLACE_WITH_CLUSTER_URL
    ```
 
-1. Apply the Kafka config:
+   **namespace-scoped**: set `namespace` to where you are planning to   create `KafkaChannel`
```

Comment:
so, there will be _serveral_ CMs - in different namespaces ?

Reply:
yes